rcmd/Rsettings.go

```diff
@@ -21,16 +22,26 @@ func NewRSettings(rPath string) RSettings {
 }
 
 // R provides a cleaned path to the R executable
-func (rs RSettings) R() string {
-	if rs.Rpath == "" {
-		return ("R")
+func (rs RSettings) R(os string) string {
+	r := "R"
+	if os == "windows" {
```

Comment:
isn't this also fixed in 41, no need to double set it

Reply:
It's needed if rs.Rpath is not empty and does not have the extension